res/css/views/settings/tabs/_SettingsTab.pcss

```diff
@@ -62,6 +62,14 @@ limitations under the License.
     margin-inline-end: 80px; /* Align with the rest of the view */
     margin-bottom: 10px; /* TODO: Use a spacing variable */
     margin-inline-start: 0;
+
+    /* Making copy button stick to the right while scrolling */
+    .mx_CopyableText_border .mx_CopyableText_copyButton {
+        position: sticky;
+        right: 0;
+        z-index: 100;
```

Comment:
are you sure setting the z-index is necessary? It seemed to work fine when I disabled it.